indico/MaKaC/common/offlineWebsiteCreator.py

```diff
@@ -251,14 +251,15 @@ def _normalize_path(self, path):
         return secure_filename(remove_tags(path))
 
     def _getAllMaterial(self):
-        self._addMaterialFrom(self._conf, "events/conference")
-        for contrib in self._conf.getContributionList():
-            self._addMaterialFrom(contrib, "agenda/%s-contribution" % contrib.getId())
-            if contrib.getSubContributionList():
-                for sc in contrib.getSubContributionList():
-                    self._addMaterialFrom(sc, "agenda/%s-subcontribution" % sc.getId())
-        for session in self._conf.getSessionList():
-            self._addMaterialFrom(session, "agenda/%s-session" % session.getId())
+        event = self._conf.as_event
+        self._addMaterialFrom(event, "events/conference")
+        for contrib in event.contributions:
```

Comment:
Not sure if that's going to happen (anytime soon) though. Especially for relationships it'd be an awful lot of magic.